packages/host/src/AppBar.tsx

```diff
@@ -1,108 +1,43 @@
-import React, { useState } from 'react';
+import React from 'react';
```

Comment:
In here I took out the Breadcrumbs component, the `ArrowIcon` I just used `sx` and the `H6` component I tried hard to understand what `flexGrow` was doing but I couldn't figure it out so I removed it so that I can add it back later when you school me.

Also the toolbar I just used `sx` for the single prop